pkg/account/downloader/internal/http/groups.go

```diff
@@ -30,7 +30,7 @@ func (c *Client) GetGroups(ctx context.Context, accUUID string) ([]accountmanage
 		return nil, err
 	}
 	if r != nil && int(r.Count) != len(r.Items) {
-		return nil, errors.New("the received data are inconsistent")
+		return nil, fmt.Errorf("the received data is inconsistent: count(%d) != items(%d)", int(r.Count), len(r.Items))
```

Comment:
I don't think "is inconsistent" tells a user anything about what's wrong or what to do. Can be log some concrete error here? I forsee all of these errors ending up at the dev team with a "what does this mean? what do I do?".